network/ipmasq_windows.go

```diff
@@ -0,0 +1,34 @@
+// +build windows
```

Comment:
Why have a windows version of this file?

Reply:
Had to move some IPMASQ functions out of main - see updated file.